sormas-app/app/build.gradle

```diff
@@ -131,7 +130,7 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
     androidTestImplementation 'androidx.test.uiautomator:uiautomator:2.2.0'
     implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0'
-    implementation 'me.dm7.barcodescanner:zxing:1.9.13'
+    implementation 'me.dm7.barcodescanner:zxing:1.9.8'
```

Comment:
What's the reason for the downgrade here? Does mavenCentral not contain 1.9.13?

Reply:
Exactly. 1.9.13 is in jcenter, but maven central has 1.9.8